gamemodes/horde/gamemode/sv_commands.lua

```diff
@@ -107,7 +119,7 @@ function Shop(ply)
     if ply:Alive() then
         local res = hook.Run("Horde_OnPlayerOpenShop", ply)
         if res ~= true then
-            if HORDE.current_break_time <= 0 then
+            if HORDE.current_break_time <= 0 and not NearBeacon(ply) then
```

Comment:
```suggestion
            if HORDE.current_break_time <= 0 and not nearBeacon(ply) then
```

Reply:
all the other functions here used the same case